### PR TITLE
Grammar.

### PR DESCRIPTION
**Pull request by JPermSolves**

### CODE REVIEW

**Comment by JPermSolves** [comment]
<p>This is over.</p>

File: index.html [hunk 1]
```diff
@@ -162,7 +162,7 @@ <h1>Standings</h1>
 		</table>
 		<div id="news">
 			<h1>News</h1>
-			<p>News:B.P.R score legendary goal against anesasa.the flyers win 6-0 2 goals from 8 year old named ternwoe he comes from stands when substutotion comes he comes from the crowd and makes it like he comes on and now ternwoe starter for the flyers.ternwoe go to urtown.</p>
+			<p>B.P.R score legendary goal against Anesasa. The Fliers win 6-0 and 2 goals came from the 8 year old named Ternwoe. He came from the stands when the substitution comes. He comes from the crowd and makes it like he comes on. Now Ternwoe is a starter for The Flyers. Ternwoe went to <span class="urtown-united">Urtown United</span>.</p>
```
Comment (HatchlingsCheeseburger) [inline index.html]: I think you should do "He came from the stands when the substitution" came instead of "comes"?

- [ ] tap when done.

Comment (HatchlingsCheeseburger) [reply index.html]: but tell me if you disagrea!

@HatchlingsCheeseburger

Comment (JPermSolves) [reply index.html]: <p><em>Done!</em></p>

**Pull request merged**